docs/artifacts/two-sessions-one-container.html

```
@@ -0,0 +1,123 @@
+<title>Two Sessions, One Container</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)